src/analysis/RDFatomistic.py

```diff
@@ -1,46 +1,60 @@
-#  Copyright (C) 2012,2013
+#  Copyright (C) 2012,2013,2014,2015,2016
 #      Max Planck Institute for Polymer Research
 #  Copyright (C) 2008,2009,2010,2011
 #      Max-Planck-Institute for Polymer Research & Fraunhofer SCAI
-#  
+#
 #  This file is part of ESPResSo++.
-#  
+#
 #  ESPResSo++ is free software: you can redistribute it and/or modify
 #  it under the terms of the GNU General Public License as published by
 #  the Free Software Foundation, either version 3 of the License, or
 #  (at your option) any later version.
-#  
+#
 #  ESPResSo++ is distributed in the hope that it will be useful,
 #  but WITHOUT ANY WARRANTY; without even the implied warranty of
 #  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 #  GNU General Public License for more details.
-#  
+#
 #  You should have received a copy of the GNU General Public License
-#  along with this program.  If not, see <http://www.gnu.org/licenses/>. 
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 r"""
 ************************************
 **espressopp.analysis.RDFatomistic**
 ************************************
 
+Class to compute radial distribution functions in adaptive resolution simulations in subregions of the box.
 
-.. function:: espressopp.analysis.RDFatomistic(system, type1, type2, _span)
+On the one hand, the RDF can be calculated in a cuboid region in the center of the box (periodic in y,z, limited in x). In this case, particle pairs are considered for which at least one of them is in the defined cuboid region (spanbase = True). This can be useful when the high resolution region has a slab geometry. No further normalization should be required. On the other hand, the routine can also calculate unnormalized RDFs using particle pairs with both particles being in the high resolution region. This can be useful when atomistic region has complicated or spherical geometries. In any case, only atoms belonging to different molecules (i.e. different coarse-grained entities) are considered.
 
-		:param system: 
-		:param type1: 
-		:param type2: 
-		:param _span: 
-		:type system: 
-		:type type1: 
-		:type type2: 
-		:type _span: 
+Examples:
+
+>>> rdf_0_1 = espressopp.analysis.RDFatomistic(system = system, type1 = 0, type2 = 1, span = 1.5, spanbased = True)
+>>> # creates the class for calculating the RDF between atomistic particles of type 1 and 0 between different molecules,
+>>> # within plus/minus 1.5 from the center of the box in x-direction
+
+>>> rdf_0_1.compute(100)
+>>> # computes the rdf using 100 bins over a distance corresponding to L_y / 2.0
+
+.. function:: espressopp.analysis.RDFatomistic(system, type1, type2, span, spanbased)
+
+                :param system: system object
+                :param type1: type of atom 1
+                :param type2: type of atom 2
+                :param span: (default: 1.0) radius of the cuboid region used for rdf calculation (if spanbased == True)
```

Comment:
Because span appears before spanbased in the list of arguments, it looks like span is always used if spanbased is used. Need to explicitly say here that span is not used if spanbased == False. Also need to say explicitly somewhere in documentation that if spanbased == False, the decision about what particles to include is based on their value of lambda_adr, i.e. as determined elsewhere in the code. Otherwise it looks like the value of span is being used to define the high resolution region when spanbased == False.